client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -142,6 +166,107 @@ class TransferDomainPrecheck extends React.PureComponent {
 		);
 	}
 
+	getRegistrationPeriodMessage() {
```

Comment:
This file is getting really long (almost 500 lines? 😱 ). Maybe, for example, this method could be a separate component instead?